src/commands/Fun/jumbo.js

```diff
@@ -0,0 +1,36 @@
+
+exports.run = function (bot, msg, args) {
+
+    if (args.length < 1) {
+        throw 'Please provide a emoji to enlarge';
+    }
+
+    if (args[0].charCodeAt(0) >= 55296) {
+        msg.delete();
+        msg.channel.send({
+            embed: bot.utils.embed(args[0], 'Can not enlarge built-in discord emoji.')
+        }).then(m => m.delete(15000));
+
+        return;
+    }
+
+    const match = args[0].match(/<:[a-zA-Z0-9_-]+:(\d{18})>/);
+
+    if (!match[1]) {
+        throw 'Please provide a valid emoji!';
+    }
+
+    const emoji = bot.emojis.get(match[1]);

```

Comment:
Check how I did this in `//emoji` — you need to make sure that `emoji` has a value. I have a simple way to do that in `//emoji`.